server/controllers/refer/new/additionalInformationController.ts

```diff
@@ -43,11 +43,10 @@ export default class NewReferralsAdditionalInformationController {
     }
   }
 
-  update(): TypedRequestHandler<Request, Response> {
+  skip(): TypedRequestHandler<Request, Response> {
```

Comment:
I'm not sure if we need a new method.  We could just use the existing `update` method, without providing a body.